pkg/eval/builtin_fn_io.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"os"
+	//	"os"
```

Comment:
I know your subsequent commit reinstated this but in general you should not leave this type of comment in the source. Its presence means that either it is no longer needed and you forgot to remove this line before sending it out for review, or it is needed and the change is incomplete. What I do when introducing this type of comment is include "WTF" in the text to flag something needing more attention: `// WTF "os"`. In the four decades I've been programming I can count on the fingers of one hand the number of times "WTF" legitimately appeared in a project. I always do a "grep WTF **" to check for the presence of such comments to verify I have resolved all such comments before creating a pull-request.

P.S., I am not suggesting you handle this situation the way I do it. You should use whatever approach helps you avoid sending a change for review while it still has unresolved issues as indicated by this type of comment.

Reply:
This is a piece of great advice thank you for this.